scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -1107,15 +1146,18 @@ void BPF_STRUCT_OPS(rusty_enqueue, struct task_struct *p, u64 enq_flags)
 	 */
 	if (!bpf_cpumask_test_cpu(scx_bpf_task_cpu(p), (const struct cpumask *)p_cpumask)) {
 		cpu = scx_bpf_pick_any_cpu((const struct cpumask *)p_cpumask, 0);
+		do_iowait_boost(p, taskc, cpu);
```

Comment:
The CPU chosen here isn't necessarily going to be chosen by `@p` -- we're just kicking the CPU to ensure that CPUs in `@p`'s domestic domain aren't all idle. It's kind of a hack and I'm sure we can do this in a better way.

In general, wdyt about just calling `do_iowait_boost()` to `rusty_running()`? That way we don't have to sprinkle the calls around, and can just invoke it in the callback that's fired just before `@p` starts running on a CPU. The other nice thing about doing this all in one place is that I think it provides a nice foundation to add more complexity to the decision of whether we'll do frequency boosting. For example, we may also want to do cpufreq boosting for interactive tasks, etc.